import-service/handlers/importFileParser.js

```diff
@@ -0,0 +1,34 @@
+const csv = require('csv-parser')
+
+const AWS = require('aws-sdk');
+const s3 = new AWS.S3({region: 'eu-central-1'});
+const BUCKET = 'nodejs-products-bucket';
+
+export const importFileParser = (event) => {
+    console.log('Lambda importFileParser invocation with event: ', event);
+    event.Records.forEach(record => {
+        console.log("in records parsing file with key=" + record.s3.object.key);
+        var objKey = record.s3.object.key;
```

Comment:
Why didn't you use const instead of var?
and also you can destructure the object:
const { key } = record.s3.object;

Reply:
java logic thinking))) for us constant something that we define one time and do not change.. If you could recommend some material to dig into Nodejs i would very appreciate, as it's too many in internet.. hard to chose something.. Thanks in advance!

Reply:
es6:
https://habr.com/ru/post/313526/
es2020 (optional):
https://habr.com/ru/company/plarium/blog/485362/

Reply:
Thank you!